Resources/doc/request_body_converter_listener.rst

```diff
@@ -91,5 +91,23 @@ The validation errors will be set on the ``validationErrors`` controller argumen
         // ...
     }
 
+You can configure the validation groups used by the validator
+via the ``validator`` option:
+
+.. code-block:: php
+
+    /**
+     * @ParamConverter("post", converter="fos_rest.request_body", options={"validator"={"groups"={"foo", "bar"})
```

Comment:
Not the same fault ! https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1200#issuecomment-156137323 :D